cnn_convertor/fpga_layer.py

```diff
@@ -995,9 +995,10 @@ def __init__(self, nodes):
                     need_copy = False
                     # ignore the last input node
                     for node_in in node.input_nodes[:-1]:
-                        if (node_in.output_dim[-1] % 8 != 0) or \
-                           (node_in.output_size !=
-                                make_align_size(node_in.output_size)):
+                        if (node_in.type is NodeType.Convolution and
```

Comment:
I think it should be `node_in.type in (NodeType.Convolution, NodeType.Pooling, NodeType.InnerProduct)`, because now InnerProduct is also run by CONV module.